.changeset/poor-rules-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cube-dev/ui-kit": patch
+---
+
+fix(preset.style): font-family fallback
```

Comment:
Please, don't write changelog as you write your commits, prefer a human-oriented style instead.